src/MAUI/Maui.Samples/Samples/Layers/Add3dTilesLayer/Add3dTilesLayer.xaml.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright 2024 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using Esri.ArcGISRuntime.Mapping;
+
+namespace ArcGIS.Samples.Add3dTilesLayer
+{
+    [ArcGIS.Samples.Shared.Attributes.Sample(
+        name: "Add 3d tiles layer",
+        category: "Layers",
+        description: "Add a layer to visualize 3D tiles data that conforms to the OGC 3D Tiles specification.",
+        instructions: "When loaded, the sample will display a scene with an `Ogc3DTilesLayer`. Pan around and zoom in to observe the scene of the `Ogc3DTilesLayer`. Notice how the layer's level of detail changes as you zoom in and out from the layer.",
+        tags: new[] { "3d tiles", "OGC", "OGC API", "layers", "scene", "service" })]
+    [ArcGIS.Samples.Shared.Attributes.OfflineData()]
+    public partial class Add3dTilesLayer
+    {
+        // Create a new elevation source from Terrain3D REST service.
+        private readonly ArcGISTiledElevationSource _elevationSource = new ArcGISTiledElevationSource
+            (new Uri("https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer"));
+
+        // Create a new scene with a dark gray basemap.
+        private readonly Scene _scene = new Scene(BasemapStyle.ArcGISDarkGray);
```

Comment:
Does the `Scene` need to be a member variable?

Could we create this in `Initialize()`?

Reply:
Agreed! We can move it to Initialize method as we are adding layers and elevation sources there. So we may want to make sure `scene` is initialized before performing operations on it.